README.md

```diff
@@ -33,118 +33,33 @@ Deploying and operating a Ceph cluster is complex because Ceph is designed to be
-    |       | 10.00GiB | virtio | /dev/disk/by-id/virtio-2171ea8f-e8a9-44c7-8 |
-    +-------+----------+--------+---------------------------------------------+
-    |       | 10.00GiB | virtio | /dev/disk/by-id/virtio-cf9c6e20-306f-4296-b |
-    +-------+----------+--------+---------------------------------------------+
-
-$ sudo microceph disk add --wipe /dev/disk/by-id/virtio-46c76c00-48fd-4f8d-9
-$ sudo microceph disk add --wipe /dev/disk/by-id/virtio-2171ea8f-e8a9-44c7-8
-$ sudo microceph disk add --wipe /dev/disk/by-id/virtio-cf9c6e20-306f-4296-b
-$ sudo microceph disk list
-    Disks configured in MicroCeph:
-    +-----+---------------+---------------------------------------------+
-    | OSD |   LOCATION    |                    PATH                     |
-    +-----+---------------+---------------------------------------------+
-    | 0   | host | /dev/disk/by-id/virtio-46c76c00-48fd-4f8d-9 |
-    +-----+---------------+---------------------------------------------+
-    | 1   | host | /dev/disk/by-id/virtio-2171ea8f-e8a9-44c7-8 |
-    +-----+---------------+---------------------------------------------+
-    | 2   | host | /dev/disk/by-id/virtio-cf9c6e20-306f-4296-b |
-    +-----+---------------+---------------------------------------------+
-
-    Available unpartitioned disks on this system:
-    +-------+----------+--------+------------------+
-    | MODEL | CAPACITY |  TYPE  |       PATH       |
-    +-------+----------+--------+------------------+
-```
-
-![Dashboard](/assets/add_osd.png)
-
-```bash
-# Adding RGW Service
-$ sudo microceph enable rgw
-# Perform IO and Check cluster status
-$ sudo microceph.ceph status
-    cluster:
-        id:     a8f9b673-f3f3-4e3f-b427-a9cf0d2f2323
-        health: HEALTH_OK
-    
-    services:
-        mon: 1 daemons, quorum host (age 12m)
-        mgr: host(active, since 12m)
-        osd: 3 osds: 3 up (since 5m), 3 in (since 5m)
-        rgw: 1 daemon active (1 hosts, 1 zones)
-    
-    data:
-        pools:   7 pools, 193 pgs
-        objects: 341 objects, 504 MiB
-        usage:   1.6 GiB used, 28 GiB / 30 GiB avail
-        pgs:     193 active+clean
-```
-
-![Dashboard](/assets/enable_rgw.png)
-
-## 👍 How Can I Contribute ?
-
-1. Checkout [MicroCeph Hacking Guide](/HACKING.md) to start building and contributing to the codebase.
-2. Excited about [MicroCeph](https://snapcraft.io/microceph) ? Join our [Stargazers](https://github.com/canonical/microceph/stargazers)
-3. Write reviews or tutorials to help spread the knowledge 📖
-4. Participate in [Pull Requests](https://github.com/canonical/microceph/pulls) and help fix [Issues](https://github.com/canonical/microceph/issues)
-
-You can also find us on Matrix @[Ubuntu Ceph](https://matrix.to/#/#ubuntu-ceph:matrix.org)
+The documentation is found in the [`docs`][docs-dir-microceph] directory. It is
+written in RST format, built with Sphinx, and published on Read The Docs:
```

Comment:
The RST format and using Sphinx for docs seems like an implementation detail, do we need to bring it up?